elevation-service.md

```diff
@@ -1,72 +1,81 @@
-| `height` | An array of height for the associated lat,lng coordinates. |
+| `shape` | The specified shape coordinates from the input request. |
+| `encoded_polyline` | The specified encoded polyline coordinates from the input request. |
+| `range_height` | The 2D array of range (x) and height (y) per input latitude, longitude coordinate. |
+| `x coordinate` | The range or distance along the input locations. It is the cumulative distance along the previous latitiude, longitude coordinates up to the current coordinate. The x-value for the first coordinate in the shape will always be 0. |
+| `y coordinate` | The height or elevation of the associated latitude, longitude pair. The height is returned as `null` if no height data exists for a given location. |
+| `height` | An array of height for the associated latitude, longitude coordinates. |
 
-### Data Sources and Known Issues
+##Data sources and known issues
 
-Currently the underlying data sources for the service are a mix of [SRTM](http://www2.jpl.nasa.gov/srtm/), [GMTED](http://topotools.cr.usgs.gov/gmted_viewer/) and [GEBCO](http://www.gebco.net/data_and_products/gridded_bathymetry_data/) DEMs. These sets provide global coverage at varying resolutions up to approximately 30 meters. It should be noted that both SRTM and GMTED zero fill oceans and other bodies of water; in these areas we use GEBCO to provide bathymetry (as well as in regions which are not coverged by SRTM and GMTED). Many other classical DEM-related issues occur in these datasets. It is not uncommon to see large variations in elevation in areas with large buildings and other such structures. We are considering how to best integrate NED and NRCAN sources, and are always looking for better datasets. If you find any data issues or can suggest any supplemental open datasets please let us know by filing an issue in [valhalla/skadi](https://github.com/valhalla/skadi).
+Currently, the underlying data sources for the service are a mix of [SRTM](http://www2.jpl.nasa.gov/srtm/), [GMTED](http://topotools.cr.usgs.gov/gmted_viewer/) and [GEBCO](http://www.gebco.net/data_and_products/gridded_bathymetry_data/) DEMs. These sets provide global coverage at varying resolutions up to approximately 30 meters. It should be noted that both SRTM and GMTED fill oceans and other bodies of water; in these areas, GEBCO provides bathymetry (as well as in regions which are not coverged by SRTM and GMTED). Many other classical DEM-related issues occur in these datasets. It is not uncommon to see large variations in elevation in areas with large buildings and other such structures. We are considering how to best integrate NED and NRCAN sources, and are always looking for better datasets. If you find any data issues or can suggest any supplemental open datasets, please let us know by filing an issue in [valhalla/skadi](https://github.com/valhalla/skadi).
```

Comment:
In my opinion, we should keep it as "zero fill" so that people know they shouldnt expect zero filled bodies of water. This is very important for drawing contour lines and, I think, needs to be accurately noted.